tools/cpp_build/libtorch/CMakeLists.txt

```diff
@@ -186,18 +195,36 @@ INSTALL(TARGETS torch
   LIBRARY DESTINATION "${TORCH_INSTALL_LIB_DIR}"
   ARCHIVE DESTINATION "${TORCH_INSTALL_LIB_DIR}")
 
-SET(TORCH_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/test_jit.cpp)
+## JIT tests
+
+SET(TORCH_JIT_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/test_jit.cpp)
 
-ADD_EXECUTABLE(test_jit ${TORCH_TEST_SRCS})
+ADD_EXECUTABLE(test_jit ${TORCH_JIT_TEST_SRCS})
 
 TARGET_LINK_LIBRARIES(test_jit torch)
 
 TARGET_INCLUDE_DIRECTORIES(test_jit PUBLIC
-  "${TORCH_SRC_DIR}/../"
   "${TORCH_SRC_DIR}/../aten/src/ATen/utils/catch/single_include"
   "${COMMON_INCLUDES}")
 
 INSTALL(TARGETS test_jit
   RUNTIME DESTINATION "${TORCH_INSTALL_BIN_DIR}"
   LIBRARY DESTINATION "${TORCH_INSTALL_LIB_DIR}"
   ARCHIVE DESTINATION "${TORCH_INSTALL_LIB_DIR}")
+
+## API tests
+
+SET(TORCH_API_TEST_SRCS ${TORCH_SRC_DIR}/csrc/api/test/test.cpp)
```

Comment:
Nooo please no tests inline with the code. Let's keep everything in `test/cpp`. `test_jit.cpp` is only a legacy thing that we shouldn't follow anymore

Reply:
Oh ok, sure. I was unsure where to put it. I will create a `tests/cpp` folder 👍